### ISSUE TITLE
Export to PC to HTML

### ISSUE THREAD

**Issue by ssveter1** [richardwilkes/gcs#210]
Was it your intent to leave the export styled as the past version?

**Comment by richardwilkes** [comment]
No. Just haven't gotten around to it yet. A GCS user has done most of the work necessary and submitted it to me -- I just haven't had the opportunity yet to review it.

**Comment by ssveter1** [comment]
Oh ok. Thanks.
